third_party/efr32_sdk/efr32_board.gni

```diff
@@ -32,9 +32,13 @@ assert(efr32_board != "", "efr32_board must be specified")
 # - BRD4170A / SLWSTK6000B / Multiband Wireless Starter Kit / 2.4GHz@19dBm, 915MHz@19dBm, 868MHz@19dBm
 # - BRD4180A / SLWSTK6006A / MG21 Module / 2.4GHz@20dBm
 
+specific_brd_defines = []
```

Comment:
We should try to avoid dropping vowels out of words to shorten them.

board_defines ?

Reply:
Sure! I did the change and I'll keep this in mind.